testsuite/classlibrary/TestProxySpace.sc

```diff
@@ -52,4 +60,42 @@ TestProxySpace : UnitTest {
 		server.quit;
 		server.remove;
 	}
+
+	test_copy_as_currentEnvironment {
+
+		var environment = currentEnvironment;
+		var proxySpace = ProxySpace(server);
+		var copySpace;
+		var family = IdentitySet.new;
+		var commonProxies;
+
+		this.bootServer(server);
+
+		proxySpace.push;
+
+		~freq = 80;
+		~hpf = { HPF.ar(WhiteNoise.ar(0.1), ~freq.kr) };
+
+		0.1.wait;
+
+		copySpace = proxySpace.copy;
+
+		proxySpace.pop;
+		proxySpace.end;
+
+
+		copySpace[\hpf].getFamily(family);
+		this.assert(family.size == 2, "there should be one parent in this case", report: true); // sanity check
+
+		commonProxies = proxySpace.envir.values.as(IdentitySet) sect: family;
+		this.assert(commonProxies.isEmpty, "proxyspace copying should refresh local environment crossreferences");
+
+		proxySpace.clear;
+		copySpace.clear;
```

Comment:
i think you forgot to quit the server at the end

Reply:
ah yes, thanks. I've refactored the boot and quit for both tests.